src/airflow/dags/dag_genetics_etl_gcp.py

```diff
@@ -0,0 +1,74 @@
+"""Generate jinja2 template for workflow."""
+from __future__ import annotations
+
+from pathlib import Path
+
+import yaml
+from airflow.decorators import task_group
+from airflow.models.dag import DAG
+from airflow.operators.empty import EmptyOperator
+from common_airflow import (
+    create_cluster,
+    delete_cluster,
+    shared_dag_args,
+    shared_dag_kwargs,
+    submit_pyspark_job,
+)
+
+SOURCE_CONFIG_FILE_PATH = Path(__file__).parent / "configs" / "dag.yaml"
```

Comment:
To be more precise, Airflow isn't able to look at files outside the directories we mounted for it in Docker. In this case, it's:
* `dags`
* `configs`
* `logs`
* `plugins`

So because both `dags` and `configs` are mounted, it can access the latter as well.

However, for all files completely outside of the mounted directories, it wouldn't be able to see them.

Reply:
Thank you for the explanation